source/documentation/contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,100 @@
+
+== Modularization backstory
+oVirt docs are modularized.
+All existing content has been replaced with content that is based on user stories and
+complies with the modularization guidelines. All future content must both
+support a user story and be modular.
+
+== How do I contribute modularized content?
+To contribute modularized content, you need to write a user story, create
+documentation modules to support the user story, and create an assembly for the
+story.
+
+== What if I don't want to write in modules?
+If you don't want to write the modules yourself but have a content change,
+write a user story, provide details to support the story, and reach out to the
+oVirt docs team.
+
+== How do I write a user story? Is there a template?
+Instead of a template, we have a series of questions for you to answer to
+create the user story. Follow the same steps if you are writing the modules
+yourself or if you plan to work with the docs team.
+
+The basic format of a user story is:
+
+----
+As a <type of user>, I want to <goal state> because <reason behind the goal>.
+----
+
+For example, "As an oVirt administrator, I want to migrate the Data Warehouse to a remote machine because I want to achieve better performance for the Engine machine."
+
+Use the following questions to guide you in providing the context for your user story and the necessary technical details to start a draft.
+You don't have to answer all of these questions, only the ones that make sense to your particular user story.
+
+=== Feature info
+* What is the feature being developed? What does it do?
+* How does it work?
+* Are there any configuration files/settings/parameters being added or modified? Are any new commands being added or modified?
+* What tools or software does the docs team need to test how this feature works? Does the docs team need to update any installed software?
+* Are there any existing blogs, Wiki posts, Kbase articles, or BZs involving this feature? Or any other existing information that may help to understand this feature?
+
+=== Customer impact
+* Who is the intended audience for this feature? Does it apply to developers, admins, users, or all of the above?
+* Why is it important for our users? Why would they want to use this feature? How does it benefit them?
+* How will the customer use it? Is there a use case?
+* How will the customer interact with this feature? Web console? REST API?
+
+=== Product info
+// * Is this feature being developed for Online? Enterprise? Dedicated? OKD? All?
+* Will this feature be rolled back to previous versions?
+* What type of plan do users need to use this feature?
+* Is it user-facing, or more behind-the-scenes admin stuff?
+* What tools or software does the docs team need to test how this feature works?
+
+== How do I write in modules?
+The full guidelines for writing modules are in the Customer Content Services (CCS)
+link:https://redhat-documentation.github.io/modular-docs/[Modular documentation reference guide].
```

Comment:
Can community members access this page?

Reply:
@RichardHoch  The idea for anything on this site is that it's open and public.

Reply:
Fixed.